### DISCUSSION TITLE
如何获得网页渲染完成时的信号?

### DISCUSSION THREAD

**Discussion by ChenKe404** [CefView/QCefView#371] [Q&A] [answered]
我知道有loadEnd信号，但它只在网页加载完成时发出。
但如果此时网页内容较多，那么可能会花一些时间去渲染网页，那么这段时间界面也还是空白的。
当页面真正渲染完毕，显示出内容时的信号我要怎么才能得到呢？

**Comment by L-Super** [comment, marked as answer]
目前没有这样的信号，可查看CEF是否有你需要的接口，如果有，那么可以自行实现

**Reply by tishion** [reply]
需要前端自己通过bridge来实现，native层没有该功能。而且使用不同的前端框架，对于“渲染完成“的定义是不同的
